StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1515SingleLineCommentMustBePrecededByBlankLine.cs

```diff
@@ -129,7 +129,9 @@ private static void HandleSyntaxTreeAnalysis(SyntaxTreeAnalysisContext context)
                 }
 
                 int triviaIndex;
-                var triviaList = TriviaHelper.GetContainingTriviaList(trivia, out triviaIndex);
+
+                // PERF: Explicitly cast to IReadOnlyList so we only box once.
+                IReadOnlyList<SyntaxTrivia> triviaList = TriviaHelper.GetContainingTriviaList(trivia, out triviaIndex);
```

Comment:
:bulb: You can eliminate boxing altogether by changing the parameters of helper methods to use `DualTriviaListHelper` instead of `IReadOnlyList<SyntaxTrivia>`.